lib/b64.js

```diff
@@ -1,28 +1,23 @@
 ;(function (exports) {
   'use strict'
 
-  var lookup = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'
+  var code = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'
+  var lookup = Array.prototype.map.call(code, function (c) { return c })
```

Comment:
This broke support for IE8. Caught by the tests in feross/buffer: https://travis-ci.org/feross/buffer/builds/108872343

Will fix.